packages/@react-stately/color/test/useHexColorFieldState.test.js

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {act, renderHook} from '@testing-library/react-hooks';
+import {Color} from '../src/Color';
+import {useHexColorFieldState} from '../';
+
+describe('useHexColorFieldState tests', function () {
+  it('should be at minimum if no value is provided', function () {
```

Comment:
Need to confirm this with the team. `NumberField` starts at 0/min when no value is provided in the props

Reply:
Would it make more sense for the field to be empty or set to the minimum by default?

Reply:
I'd prefer the input to be empty before you interact with it. Then once you focus on it the first time, it'll show the minimum value if there's no value/default set. I was hoping someone from our UX team could confirm what the behavior should be

Reply:
@snowystinger 
Should start empty unless a defaultValue or value is provided.